deployment/aws-python-storage-stack-automation/storage_stack.yaml

```diff
@@ -0,0 +1,314 @@
+          - Effect: Allow
+            Action:
+              - "iam:GetRole"
+              - "iam:AttachRolePolicy"
+              - "iam:CreateRole"
+              - "iam:PutRolePolicy"
+              - "iam:DetachRolePolicy"
+              - "iam:DeleteRole"
+              - "iam:PassRole"
+              - "iam:DeleteRolePolicy"
+              - "iam:GetRolePolicy"
+              - "logs:CreateLogGroup"
+              - "logs:CreateLogStream"
+              - "logs:PutLogEvents"
+              - "logs:DescribeLogGroups"
+              - "logs:PutRetentionPolicy"
+              - "s3:PutBucketTagging"
+              - "s3:PutObject"
+              - "s3:GetObject"
+              - "s3:GetBucketTagging"
+              - "s3:ListBucket"
+              - "s3:PutEncryptionConfiguration"
+              - "s3:GetObjectTagging"
+              - "s3:ListAllMyBuckets"
+              - "s3:CreateBucket"
+              - "s3:PutObjectTagging"
+              - "sts:AssumeRole"
+              - "secretsmanager:GetSecretValue"
+              - "cloudformation:DescribeStacks"
+              - "cloudformation:GetTemplate"
+              - "cloudformation:GetTemplateSummary"
+              - "cloudformation:CreateStack"
+              - "cloudformation:ListStacks"
+              - "cloudformation:DescribeStackInstance"
+              - "cloudformation:ListStackResources"
+              - "cloudformation:DescribeStackEvents"
+              - "cloudformation:DescribeStackResource"
+              - "cloudformation:ListExports"
+              - "cloudformation:ListStackInstances"
+              - "lambda:GetLayerVersion"
+              - "lambda:PublishLayerVersion"
+              - "lambda:CreateFunction"
+              - "lambda:GetFunctionConfiguration"
+              - "lambda:DeleteFunction"
+              - "lambda:InvokeFunction"
+              - "lambda:GetEventSourceMapping"
+              - "lambda:CreateEventSourceMapping"
+              - "lambda:GetFunction"
+              - "lambda:AddPermission"
+              - "sqs:AddPermission"
+              - "sqs:GetQueueUrl"
+              - "sqs:DeleteMessage"
+              - "sqs:ReceiveMessage"
+              - "sqs:ListQueueTags"
+              - "sqs:SetQueueAttributes"
+              - "sqs:ListQueues"
+              - "sqs:GetQueueAttributes"
+              - "sqs:CreateQueue"
+              - "sqs:ListDeadLetterSourceQueues"
+              - "sqs:SendMessage"
+              - "sns:CreateTopic"
+              - "sns:Publish"
+              - "sns:SetTopicAttributes"
+              - "sns:Subscribe"
+              - "sns:GetTopicAttributes"     
+            Resource: "*"
+      Roles:
+        - !Ref lambdarole
+  storagelambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: python3.8
+      Timeout: 500
+      Handler: index.lambda_handler
+      Role: !GetAtt
+        - lambdarole
+        - Arn
+      Code:
+        ZipFile: |
+          import json
+          import logging
```

Comment:
Isn't print() enough?

Reply:
removed logger import
switched everything to print()